crates/bevy_math/src/cubic_splines.rs

```diff
@@ -182,10 +192,11 @@ impl<P: Point> CubicGenerator<P> for CubicHermite<P> {
 /// The curve passes through every control point.
 ///
 /// ### Tangency
-/// Automatically defined at each control point.
+/// Tangents are automatically computed based on the position of control points.
```

Comment:
```suggestion
/// Tangents are automatically computed based on the positions of control points.
```
I think this would be better